x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -110,17 +110,18 @@ export const defineGetBenchmarksRoute = (router: CspRouter) =>
         const cspContext = await context.csp;
 
         try {
-          const cspPackagePolicies = await getCspPackagePolicies(
+          const packagePolicies: ListResult<PackagePolicy> = await getCspPackagePolicies(
             cspContext.soClient,
             cspContext.packagePolicyService,
             CLOUD_SECURITY_POSTURE_PACKAGE_NAME,
             request.query,
-            POSTURE_TYPE_ALL
+            POSTURE_TYPE_ALL,
+            true
```

Comment:
nit: for better readability, we can introduce the `options` param, with `excludeVulnMgmtPackages: true` being an option. Otherwise, it's quite hard to understand what this `true` param means without checking out the code